docs/upgrade/phpstan-level-1.md

```diff
@@ -0,0 +1,65 @@
+# Upgrade Instructions for Upgrading PHPStan to Level 1
+
+This article describes the upgrade instructions for [#1040 Upgrading PHPStan](https://github.com/shopsys/shopsys/pull/1040).
+Pull request updates PHPStan in framework to level 4, but recommended level for your project is 1 as level 4 does not comply with framework extensibility options.
+
+Upgrade instructions are in a separate article because there is a lot of instructions and we don't want to jam the [UPGRADE-v7.3.0](/docs/upgrade/UPGRADE-v7.3.0.md).
+Instructions are meant to be followed when you upgrade from `v7.2.0` to `v7.3.0`.
+We recommend upgrading to level 1 to automatically check for better code quality.
```

Comment:
There should be more info why 1 and not higher...

Reply:
removed the sentence completely as it's explained higher and it was duplicite